# mirna-csa-hk2
## Integrative analysis of miRNA:mRNA interactome in CsA-induced nephrotoxicity
Previously, we published an integrative analysis of miRNA:mRNA interactions in a cell model of cyclosporine A induced nephrotoxicity (C. Benway, J. Iacomini, American Journal of Transplantation, 2018, [doi: 10.1111/ajt.14503](https://www.ncbi.nlm.nih.gov/pubmed/28925592)):
>Calcineurin inhibitors induce nephrotoxicity through poorly understood mechanisms thereby limiting their use in transplantation and other diseases. Here we define a microRNA (miRNA)-messenger RNA (mRNA) interaction map that facilitates exploration into the role of miRNAs in cyclosporine-induced nephrotoxicity (CIN) and the gene pathways they regulate. Using photoactivatable ribonucleoside-enhanced crosslinking and immunoprecipitation (PAR-CLIP), we isolated RNAs associated with Argonaute 2 in the RNA-induced silencing complex (RISC) of cyclosporine A (CsA) treated and control human proximal tubule cells and identified mRNAs undergoing active targeting by miRNAs. CsA causes specific changes in miRNAs and mRNAs associated with RISC, thereby altering post-transcriptional regulation of gene expression. Pathway enrichment analysis identified canonical pathways regulated by miRNAs specifically following CsA treatment. RNA-seq performed on total RNA indicated that only a fraction of total miRNAs and mRNAs are actively targeted in the RISC, indicating that PAR-CLIP more accurately defines meaningful targeting interactions. Our data also revealed a role for miRNAs in calcineurin-independent regulation of JNK and p38 MAPKs caused by targeting of MAP3K1. Together, our data provide a novel resource and unique insights into molecular pathways regulated by miRNAs in CIN. The gene pathways and miRNAs defined may represent novel targets to reduce calcineurin induced nephrotoxicity.

The purpose of this repository is to explore various bioinformatics tools available to analyze PAR-CLIP miRNA:mRNA interactome data integrated with corresponding miRNA and mRNA expression data. We will use our published HK-2 cell data in order to reproduce the findings and expand our analysis using a network approach (data accessible at NCBI GEO database (Edgar et al., 2002), accession [GSE98670](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98670)). 

## Background information
- HK-2 (human kidney 2, [ATCC® CRL-2190™](https://www.atcc.org/products/all/CRL-2190.aspx)) is a proximal tubular cell (PTC) line derived from normal kidney, immortalized by transduction with human papilloma virus 16 (HPV-16) E6/E7 genes.
- Ryan et al.(Kidney Int. 1994; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/8127021)) describes the establishment of the HK-2 cell line and evaluates its phenotype and functional characteristics.
- Previously in the Iacomini lab, two studies examined the signatures of microRNA expression in mice after renal ischemia-reperfusion injury:
  - Godwin et al. (PNAS 2010; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/20651252)) identified nine miRNAs that were differentially expressed following IRI compared to sham controls. They further demonstrated that these effects were lymphocyte-independent and explored the potential protective role of miR-21 upregulation in tubular epithelial cells following renal injury. 
  - Shapiro et al. (PLoS One 2011; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/21887224)) evaluated whether the microRNA expression signatures (571 miRNAs, 8 timepoints) could be used to define a biomarker of of renal IRI.
- Yuan et al. (AJT 2015; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/25854542) examined the role of microRNAs in CsA-induced nephrotoxicity. Using an *in vivo* mouse model and human cell lines the authors demonstrated that CsA treatment induces miR-494 expresission in proximal tubule epithelial cells (PTECs) which targets and inhibits *PTEN* expression. Blockade of miR-494/*PTEN* targeting prevented CsA-induced epithelial-to-mesenchymal transition (EMT).  
- Chen et al. (Transpl Int. 2015; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/25266172))
- Gooch et al. (PLoS One 2017; [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/28414804))

## List of data files
- [hsa_mature_dna.fasta](https://github.com/cbenway/mirna-csa-hk2/blob/master/hsa_mature_dna.fasta) - FASTA file of all mature miRNA sequences in DNA format from mirBase 21 (2015). For use with [SeedVicious v1.1](https://seedvicious.essex.ac.uk/) prediction of miRNA target sites.

## Differential expression of mRNA and miRNA in CsA-treated HK-2 human kidney proximal tuble cells
Recently, [BioJupies](https://amp.pharm.mssm.edu/biojupies/) was developed by the [Ma-ayan Lab](http://icahn.mssm.edu/research/labs/maayan-laboratory) @ the Icahn Schoold of Medicine at Mount Sinai ([paper](pubmed...) ;[source code](https://github.com/MaayanLab/biojupies)). This tool provides super fast differential expression reports on GEO data or user-uploaded FASTQs or count data. To evaluate the tool and compare to my own published analysis I produced reports for [mRNA](https://amp.pharm.mssm.edu/biojupies/notebook/UyVNJKXtV) and [miRNA](https://amp.pharm.mssm.edu/biojupies/notebook/6ohGMu6LB) differential expression in HK-2 cells treated with Cyclosporine A.
1. In our study, we reported 7,688 differentially expressed mRNAs (Benjamini-Hochberg FDR < 0.05) and 72 differentially expressed miRNAs (Benjamini-Hochberg FDR < 0.10).
  >Small RNA reads were adapter-trimmed and filtered by size (greater than 16 nucleotides) using ‘Fastx_clipper’ (HannonLab 2014). Trimmed reads were aligned to the ‘hg19canon’ reference genome using Bowtie [-l 17 -n 0 -k 1 -m 100 -best -strata]
(Langmead et al. 2009). Counts of reads mapped to mature miRNAs (miRBase v21) were then computed using the count feature of HTSeq (Anders, Pyl, and Huber 2015). Count matrices were used to detect differentially expressed genes using the R package ‘DESeq2’ (Michael I Love, Huber, and Anders 2014). A similar workflow was utilized for mRNA differential expression analysis, except that TopHat (Trapnell et al. 2012) was utilized for alignment and a GFF reference containing the hg19 coordinates for all gene exons was used to compute count matrices.
2. In the BioJupies notebook analysis, we observe 8,910 differentially expressed mRNAs (BH adjusted P value < 0.05) and 65 differentially expressed miRNAs (BH adjusted P value < 0.10).
  > The GEO dataset GSE98670 is loaded into the notebook. Expression data was quantified as gene-level counts using the ARCHS4 pipeline (Lachmann et al. 2017), available [here](http://amp.pharm.mssm.edu/archs4/). For the miRNA expression analysis, the same count matrix as above was used as input. BioJupies uses a limma-voom framework to compute differential gene expression, DGE.
